src/cbmc/cbmc_parse_options.cpp

```diff
@@ -222,6 +222,10 @@ void cbmc_parse_optionst::get_command_line_options(optionst &options)
   if(cmdline.isset("nondet-static"))
     options.set_option("nondet-static", true);
 
+  if(cmdline.isset("nondet-static-matching"))
```

Comment:
Should we not add a test that covers this?

Reply:
This change ought to be removed. The option should be in goto-instrument only.

Reply:
@jimgrundy, removed.

Reply:
Thanks